### ISSUE TITLE
Relations widget fails accessibility tests in CI

### ISSUE THREAD

**Issue by thatbudakguy** [geoblacklight/geoblacklight#1506]
### Descriptive summary

The aXe a11y tests that run in CI are reporting color contrast errors stemming from the relations widget:

```
1) color-contrast: Elements must meet minimum color contrast ratio thresholds (serious)
           https://dequeuniversity.com/rules/axe/4.9/color-contrast?application=axeAPI
           The following 12 nodes violate this rule:
           
               Selector: .relationship-member_of_ancestors > .card-header > h2
               HTML: <h2 class="mb-0 h6">Belongs to collection...</h2>
               Fix any of the following:
               - Element has insufficient color contrast of 1.16 (foreground color: #eceded, background color: #fefefe, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1
               
               Selector: .relationship-member_of_ancestors > .list-group.list-group-flush > .border-bottom-0.list-group-item > a[href$="the-related-record"]
               HTML: <a href="/catalog/the-related-record">
               Fix any of the following:
               - Element has insufficient color contrast of 1.11 (foreground color: #ebf3ff, background color: #ffffff, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1
               
               Selector: .relationship-part_of_ancestors > .card-header > h2
               HTML: <h2 class="mb-0 h6">Is part of...</h2>
               Fix any of the following:
               - Element has insufficient color contrast of 1.16 (foreground color: #eceded, background color: #fefefe, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1
               
               Selector: .relationship-part_of_ancestors > .list-group.list-group-flush > .border-bottom-0.list-group-item > a[href$="the-related-record"]
               HTML: <a href="/catalog/the-related-record">
               Fix any of the following:
               - Element has insufficient color contrast of 1.11 (foreground color: #ebf3ff, background color: #ffffff, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1
               
               Selector: .relationship-relation_ancestors > .card-header > h2
               HTML: <h2 class="mb-0 h6">Related records...</h2>
               Fix any of the following:
               - Element has insufficient color contrast of 1.16 (foreground color: #eceded, background color: #fefefe, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1
               
               Selector: .relationship-relation_ancestors > .list-group.list-group-flush > .border-bottom-0.list-group-item > a[href$="the-related-record"]
               HTML: <a href="/catalog/the-related-record"><span class="icon-missing geoblacklight-none"></span>The Related Record</a>
               Fix any of the following:
               - Element has insufficient color contrast of 1.11 (foreground color: #ebf3ff, background color: #ffffff, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1
               
               Selector: .relationship-replaces_ancestors > .card-header > h2
               HTML: <h2 class="mb-0 h6">Replaces...</h2>
               Fix any of the following:
               - Element has insufficient color contrast of 1.16 (foreground color: #eceded, background color: #fefefe, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1
               
               Selector: .relationship-replaces_ancestors > .list-group.list-group-flush > .border-bottom-0.list-group-item > a[href$="the-related-record"]
               HTML: <a href="/catalog/the-related-record"><span class="icon-missing geoblacklight-none"></span>The Related Record</a>
               Fix any of the following:
               - Element has insufficient color contrast of 1.11 (foreground color: #ebf3ff, background color: #ffffff, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1
               
               Selector: .relationship-source_ancestors > .card-header > h2
               HTML: <h2 class="mb-0 h6">Source records...</h2>
               Fix any of the following:
               - Element has insufficient color contrast of 1.16 (foreground color: #eceded, background color: #fefefe, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1
               
               Selector: .relationship-source_ancestors > .list-group.list-group-flush > .border-bottom-0.list-group-item > a[href$="the-related-record"]
               HTML: <a href="/catalog/the-related-record">
               Fix any of the following:
               - Element has insufficient color contrast of 1.11 (foreground color: #ebf3ff, background color: #ffffff, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1
               
               Selector: .relationship-version_of_ancestors > .card-header > h2
               HTML: <h2 class="mb-0 h6">Is version of...</h2>
               Fix any of the following:
               - Element has insufficient color contrast of 1.16 (foreground color: #eceded, background color: #fefefe, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1
               
               Selector: .relationship-version_of_ancestors > .list-group.list-group-flush > .border-bottom-0.list-group-item > a[href$="the-related-record"]
               HTML: <a href="/catalog/the-related-record">
               Fix any of the following:
               - Element has insufficient color contrast of 1.11 (foreground color: #ebf3ff, background color: #ffffff, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1
```

See e.g. https://github.com/geoblacklight/geoblacklight/actions/runs/10375337056/job/28725735086?pr=1503.
